data/sql/updates/pending_db_world/rev_1571346137981297600.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1571346137981297600');
+
+UPDATE `creature_template` SET `unit_flags` = 256 WHERE (`entry` = 35984);
```

Comment:
In the future, there should be a new line at the end of the file

Reply:
I added a newline and removed the parentheses, they are not needed here.

Reply:
Thanks for the feedback on my first PR!